CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [5.4.2]
+
+### Changed
+- calculate the indices of pixel in MAP space based on the indices and height of the pixel in SAR space
+- improve the method of determination of the reference pixel in SAR space
```

Comment:
It might be nice to explain how the method has been improved.

Reply:
where should I put the explanation?